server/cmd/mattermost/main.go

```diff
@@ -17,7 +17,6 @@ import (
 
 	// Blank imports for each product to register themselves
 	_ "github.com/mattermost/mattermost/server/v8/boards/product"
```

Comment:
We're still initializing Boards product?

Reply:
Yes, same thing. We did remove the `init()` method, so that import is not initializing the product anymore, and it will be removed in that same follow up PR